### ISSUE TITLE
Using iamges that have been normalized already

### ISSUE THREAD

**Issue by abbasmazrouei** [sanghyun-son/EDSR-PyTorch#335]
I want to use your pre-trained model's algorithm on images that have been normalized by0-1 already and I like the output to be in this range (0-1) too and not be 0-255.
so which sections should I change?
